services/webhook/elder_eval/harvest_bot_findings.py

```
@@ -0,0 +1,389 @@
+anchor this join). One row vocabulary for all three sources: ``{source, repo,
+pr, path, line, severity, category, finding, outdated, ts, url}``;
+``severity`` normalized to low/medium/high/critical.
+"""
+
+from __future__ import annotations
+
+import json
+import logging
+import re
+from dataclasses import asdict, dataclass
+from typing import Any, Iterable
```

Ruff (0.15.20)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)